lib/ref_arch_setup/bolt_helper.rb

```diff
@@ -91,6 +93,61 @@ def self.run_task_with_bolt(task, params, nodes)
       return success
     end
 
+    # Run a plan with bolt on given nodes
+    #
+    # @author Sam Woods
+    #
+    # @param task [string] Task to run on nodes
+    # @param params [hash] task parameters to send to bolt
+    # @param nodes [string] Host or space delimited hosts to run task on
+    # @param modulepath [string] The modulepath to use when running bolt
+    #
+    # @return [string] The output from the bolt run
+    def self.run_plan_with_bolt(plan, params, nodes, modulepath = RAS_MODULE_PATH)
```

Comment:
Could we just merge run plan and run task into one somehow?  They are like 90% the same code.

Reply:
'(SLV-240) RAS - refactor bolt command calls' has been created to address this issue